hbase-common/src/main/java/org/apache/hadoop/hbase/io/encoding/RowIndexSeekerV1.java

```diff
@@ -359,26 +361,30 @@ public Cell toCell() {
         // TODO : reduce the varieties of KV here. Check if based on a boolean
         // we can handle the 'no tags' case.
         if (tagsLength > 0) {
+          // TODO : getRow len here.
```

Comment:
Its ok not to get here but leave that to be decoded within SizeCachedKeyValue.  So we can remove this TODO  IMO